documentation/model.md

```diff
@@ -1438,32 +1479,37 @@ var model = new falcor.Model({cache: {
     }
 }});
 
-model.
-    treatErrorsAsValues().
-    get('titlesById[44].name').
-    then(
-        response => {
-            console.log(JSON.stringify(response.json, null, 4));
-        },
-        pathValue => {
-            // not called!
-            console.error(JSON.stringify(pathValue));
-        });
+var response;
+try {
+    response = await model.
+        treatErrorsAsValues().
+        get('titlesById[44].name');
+} catch (error) {
+    console.log('error was thrown');
+    error
```

Comment:
Can we make sure to add return statements if the intention is to add data to the console? At the moment this looks like a noop or a bug?

Reply:
done